### PR TITLE
Explaining the use of DXF import to define a complex board shape.

### PR DESCRIPTION
**Pull request by MarcoHess**
Here is some new pcbnew documentation to explain DXF import for complex board shapes and a few notes around the DXF format for successful import.

### CODE REVIEW

File: src/Pcbnew/Pcbnew_create_and_modify_board.adoc [hunk 1]
```diff
@@ -34,6 +34,47 @@ The resulting outline might look something like this:
...
+Notes on DXF imports
+^^^^^^^^^^^^^^^^^^^^
+
+There are some limitations on the DXF formats that are supported in KiCad. 
+
+For example there is currently no support for importing Ellipses. However it 
+is often possible to export the DXF from your CAD system in older DXF formats. 
+For example the DXF R12 format does not have support for Ellipses line 
+segments. As such CAD application exports Ellipses as a series of small line
+segments to 'simulate' an Ellipses. LibreCAD for example is known to do that.
```
Comment (nickoe) [inline src/Pcbnew/Pcbnew_create_and_modify_board.adoc]: I think simulate should be approximate.

File: src/Pcbnew/Pcbnew_create_and_modify_board.adoc [hunk 2]
```diff
@@ -34,6 +34,47 @@ The resulting outline might look something like this:
 
 image:images/Pcbnew_simple_board_outline.png[]
 
+Importing a DXF drawing for a board outline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As an alternative to drawing the board outline in Pcbnew directly, a 
+board outline can also be imported from a DXF drawing. This can be used to 
+create more complex board or allows board shapes to be defined in other
```
Comment (nickoe) [inline src/Pcbnew/Pcbnew_create_and_modify_board.adoc]: there is something about the board(s) and s' that is not right here. Please fix.

**Comment by MarcoHess** [comment]
Some updated wording based on the comments.

**Comment by nickoe** [comment]
Thank you for your contributions.

**Pull request merged**